src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -74,6 +75,16 @@ public void execute_multipleKeywords_multiplePersonsFound() {
         assertEquals(Arrays.asList(CARL, ELLE, FIONA), model.getFilteredPersonList());
     }
 
```

Comment:
Need to do unit test for more scenarios: 
Person has no car finding Car VRN, 
Person has No Car finding Car VRN + Name, 
Person has Car finding Car VRN + Name, 
Person has Car finding Name,
Person has Car finding Multiple Car VRN
Person with car empty keywords,
+ any more senarios, the more the better